Tools/cases_generator/generate_cases.py

```diff
@@ -168,9 +168,14 @@ def __init__(self, inst: parser.InstDef):
                 break
         self.unmoved_names = frozenset(unmoved_names)
         if self.register:
-            fmt = "IBBB"
+            num_regs = len(self.input_effects) + len(self.output_effects)
+            num_dummies = (num_regs // 2) * 2 + 2 - num_regs
+            fmt = "I" + "B"*num_regs + "_"*num_dummies
```

Comment:
I'm not sure I follow what this is doing... I would think that the goal is to sometimes add a single dummy byte? But this seems to always add either one or two dummy bytes.

Reply:
Good call, that's a typo. Since I don't have unit tests for metadata generation yet I tried this formula by hand, and then mis-transcribed it. the last `2` should be `1`.